rb_tree.py

```diff
@@ -9,48 +9,55 @@
 
 
 class Node:
+
     def __init__(self, value, color, parent, left=None, right=None):
+        
         self.value = value
         self.color = color
         self.parent = parent
         self.left = left
         self.right = right
 
     def __repr__(self):
+        
         return '{color} {val} Node'.format(color=self.color, val=self.value)
 
     def __iter__(self):
+        
+        print(str(self.value) + " (" + str(self.color)+")")
```

Comment:
I don't think we should have the print statement be in the `__iter__()` method. You do not always want to print

Reply:
Sure! It was part of some tests that I was doing. Now, it was deleted. Thanks!